src/vm/vars.hpp

```diff
@@ -341,7 +341,7 @@ class REF : public OBJECTREF
 
 };
 
-#define VALIDATEOBJECTREF(objref) ((objref).Validate())
+#define VALIDATEOBJECTREF(objref) (if (objref != NULL) (objref).Validate())
 #define VALIDATEOBJECT(obj) obj->Validate()
```

Comment:
Don't we need the null validation here too? I think I have seen one place where you have added the check in your previous iteration before a call to the VALIDATEOBJECT.